config/secrets.yml.template

```diff
@@ -10,17 +10,20 @@
 # Make sure the secrets in this file are kept private
 # if you're sharing your code publicly.
 
-development: &dev
+default: &default
```

Comment:
Please add a development entry such that it just works, when you copy secrets.yml.template to config/secrets.yml and start working in development mode.